token/js/src/instructions/decode.ts

```diff
@@ -120,6 +120,13 @@ export function isInitializeMintInstruction(decoded: DecodedInstruction): decode
     return decoded.data.instruction === TokenInstruction.InitializeMint;
 }
 
+/** TODO: docs */
+export function isInitializeMint2Instruction(
+    decoded: DecodedInstruction
+): decoded is DecodedInitializeMint2Instruction {
+    return decoded.data.instruction === TokenInstruction.InitializeMint2;
+}
+
```

Comment:
nit: the order of these in all the places in this file is intentional, following https://github.com/solana-labs/solana-program-library/blob/master/token/js/src/instructions/types.ts

Reply:
Gotcha! I'll fix this

Reply:
fixed!